src/PowerShellEditorServices.Hosting/Internal/NamedPipeUtils.cs

```diff
@@ -111,8 +109,7 @@ public static string GenerateValidNamedPipeName(IReadOnlyCollection<string> pref
                 {
                     return pipeName;
                 }
-
-            } while (tries < 10);
```

Comment:
This was never incrementing tries and therefore had actually been an infinite loop.